### PR TITLE
Add iris_rplidar model

### PR DESCRIPTION
**Pull request by jgoppert**
Add support for iris with rplidar.

### CODE REVIEW

**Comment by jgoppert** [comment]
``bash
jgoppert@jmg-bnb1:~/git/px4/catkin$ roslaunch px4 mavros_posix_sitl.launch vehicle:=iris_rplidar est:=lpe
```

![image](https://cloud.githubusercontent.com/assets/473772/22839148/4c3b05dc-ef96-11e6-970a-79f7381b1cbf.png)

File: src/firmware/posix/sitl_target.cmake [hunk 1]
```diff
@@ -69,7 +69,7 @@ set_target_properties(sitl_gazebo PROPERTIES EXCLUDE_FROM_ALL TRUE)
 # create targets for each viewer/model/debugger combination
 set(viewers none jmavsim gazebo replay)
 set(debuggers none ide gdb lldb ddd valgrind)
-set(models none iris iris_opt_flow standard_vtol plane solo tailsitter typhoon_h480)
+set(models none iris iris_hokuyo iris_opt_flow standard_vtol plane solo tailsitter typhoon_h480)
```
Comment (jgoppert) [inline src/firmware/posix/sitl_target.cmake]: Ooops, that needs to be cleaned up.

Comment (mhkabir) [inline src/firmware/posix/sitl_target.cmake]: I'll fix it and push here after testing

**Comment by jgoppert** [comment]
I'm currently just building all of this within a ros catkin workspace, so Iam not using ninja right now. I have used it in the past.

**Comment by cserko** [comment]
I'm using rplidar model with ROS and gazebo, when I look at the topics I see laser/scan topic, however, it does not have a mavros prefix how can I change for that? 
Also, when I echo  the laser/scan topic I see -inf all the time even there is an obstacle right next to iris

**Pull request merged**